src/test/java/com/snowflake/kafka/connector/SinkTaskIT.java

```diff
@@ -323,4 +328,126 @@ public void testMultipleSinkTasksWithLogs() throws Exception {
     assert offsetMap1.get(topicPartitions0.get(0)).offset() == BUFFER_COUNT_RECORDS_DEFAULT;
     assert offsetMap0.get(topicPartitions1.get(0)).offset() == BUFFER_COUNT_RECORDS_DEFAULT;
   }
+
+  @Test
+  public void testTopicToTableMapParseAndCreation() {
+    // TODO @rcheng question: should i duplicate this test for streaming?
```

Comment:
Should I duplicate this test in SnowflakeSinkTaskForStreamingIt? I didn't because i don't think this parsing and table name generation will differ across streaming vs snowpipe.

Reply:
Might be worth adding it to only `SnowflakeSinkTaskForStreamingIt` but this is fine as well. probably in future we might get away with snowpipe code but not streaming and it is easy enough to forget adding it in streaming but removing it from here.

Reply:
Thats a good point, thanks! I created a static runner inside streamingIt that is called from sinkTaskIt. Opened a different thread about best practices for that, so resolving this thread